### PR TITLE
Polish blueprint mode of pre-chat starter prompts

### PR DESCRIPTION
**Pull request by compulim**
### Added

-  Added blueprint of starter prompts in the Fluent theme, in PR [#5270](https://github.com/microsoft/BotFramework-WebChat/pull/5270), [#5276](https://github.com/microsoft/BotFramework-WebChat/pull/5276) and [#5279](https://github.com/microsoft/BotFramework-WebChat/pull/5279), by [@OEvgeny](https://github.com/OEvgeny) and [@compulim](https://github.com/compulim)

## Description

This is continuation of #5276.

Polishes the blueprint of starter prompts and supports RTL in animation.

## Specific Changes

- Corrected gaps between skeleton of title and subtitle
- Updated animation to support RTL
- Also fixed flakiness test related to upload
   - After upload and send, should wait for outgoing activities to be sent

-  [x] I have added tests and executed them locally
-  [x] I have updated `CHANGELOG.md`
-  [x] ~I have updated documentation~

## Review Checklist

> This section is for contributors to review your work.

-  [x] ~Accessibility reviewed (tab order, content readability, alt text, color contrast)~
-  [x] ~Browser and platform compatibilities reviewed~
-  [x] CSS styles reviewed (minimal rules, no `z-index`)
-  [x] ~Documents reviewed (docs, samples, live demo)~
-  [x] ~Internationalization reviewed (strings, unit formatting)~
-  [x] ~`package.json` and `package-lock.json` reviewed~
-  [x] ~Security reviewed (no data URIs, check for nonce leak)~
-  [x] Tests reviewed (coverage, legitimacy)

### CODE REVIEW

File: packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css [hunk 1]
```diff
@@ -71,12 +77,14 @@
   &:empty::after {
     border-radius: 16px;
     height: 16px;
+    margin-top: 4px;
```
Comment (OEvgeny) [inline packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css:80]:
```suggestion
    margin-block-start: var(--webchat-spacingVerticalXS);
```

Comment (OEvgeny) [reply packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css:80]: Actually should specify gap on line 27
```suggestion
```

File: packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css [hunk 2]
```diff
@@ -71,12 +77,14 @@
...
     width: 100%;
   }
 
   &:empty::before {
     border-radius: 18px;
     height: 18px;
+    margin-bottom: 2px;
```
[the remark this answers is not in the document]
Comment (OEvgeny) [reply packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css:87]: Actually should specify gap on line 27
```suggestion
```

File: packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css [hunk 3]
```diff
@@ -44,18 +44,24 @@
...
-  @media (prefers-reduced-motion: no-preference) {
+  @media (prefers-reduced-motion: reduce) {
     &:empty::after,
     &:empty::before {
-      animation-duration: 3s;
-      animation-iteration-count: infinite;
-      animation-name: blueprintAnimation;
-      animation-timing-function: linear;
+      animation-delay: -1s !important;
+      animation-play-state: paused !important;
```
Comment (OEvgeny) [inline packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css:51]:
```suggestion
      animation-delay: -1s;
      animation-play-state: paused;
```
Why raising importance like this?

Comment (OEvgeny) [reply packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css:51]: I'm ok to have it in tests, but the rule for CSS should be avoid `!important` at any costs

Comment (compulim) [reply packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.module.css:51]: Because I was just copying from yours. 🤣

**Review by OEvgeny — APPROVED** [review]

**Review by OEvgeny — APPROVED** [review]

**Review by OEvgeny — COMMENTED** [review]
After some thinking... We should adjust:
```css
  &[aria-disabled='true'],
  &:empty {
    cursor: default;
    row-gap: 14px;
  }
```
to either:
```css
  &[aria-disabled='true'],
  &:empty {
    cursor: default;
  }
  &:empty {
    row-gap: 14px;
  }
  ```
  Or update the `row-gap` for the whole card UX

**Pull request merged**